demos/signature/EVP_Signature_demo.c

```diff
@@ -218,11 +215,11 @@ int main(void)
         fprintf(stderr, "OSSL_LIB_CTX_new() returned NULL\n");
         goto cleanup;
     }
-    if (demo_sign(libctx, sig_name, &sig_len, &sig_value) != 1) {
+    if (demo_sign(libctx, sig_name, &sig_len, &sig_value) <= 0) {
         fprintf(stderr, "demo_sign failed.\n");
         goto cleanup;
     }
-    if (demo_verify(libctx, sig_name, sig_len, sig_value) != 1) {
+    if (demo_verify(libctx, sig_name, sig_len, sig_value) <= 0) {
```

Comment:
!demo_verify()

Reply:
Done in latest upload, thanks for your comments.